Mdominykas/Encountify
# Used logging, middleware and interceptor

Used logging, middleware and interceptor in EncountifyAPI.

Not sure if this "logs most of important logic" so any ideas would be helpful. 

Testing:
 - Launch EncountifyAPI project with IIS Express.
 - Execute some get/post/delete command.
 - Check your logged responses in "/EncountifyAPI/log-.txt"
